spec/integration/user_show_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Specific User', type: :feature do
+  describe 'Show Page' do
+    before(:each) do
+      @user1 = User.create(name: 'Usom', photo: 'https://source.unsplash.com/user/c_v_r/100x100', bio: 'Usom bio')
```

Comment:
##### Dear Team,. for a user_show route some tests are missing please consider adding them. 
- [ ] Tests that assert you can see the number of posts the user has written.
- [ ] Tests that assert you can see a button that lets me view all of a user's posts.
- [ ] Tests that redirect to the post's show page when you click on a user's post.
- [ ] Tests that redirects to the user's post's index page when you click on a **see all posts**.